src/api/app/controllers/webui/request_controller.rb

```diff
@@ -573,6 +573,10 @@ def prepare_request_data
     handle_notification
   end
 
+  def review_by_staging_project(review)
+    review.by_project && Project.find_by(id: review.project_id).staging_workflow_id
+  end
```

Comment:
I haven't checked, but it looks like we could use `staging_project?`...: https://github.com/openSUSE/open-build-service/blob/0e6498f3c6fe3d1ace26eb8f8f3a929689c24f7b/src/api/app/models/review.rb#L302-L304

Reply:
Nice catch, thanks. I changed to use it